official/wide_deep/wide_deep.py

```diff
@@ -216,6 +216,20 @@ def eval_input_fn():
         flags.stop_threshold, results['accuracy']):
       break
 
+  # Export the model
+  if flags.export_dir is not None:
+    wide_columns, deep_columns = build_model_columns()
```

Comment:
Maybe separate out into a separate function to keep the main a little cleaner? Then just call the fn if export_dir is not None.